aries-site/src/examples/templates/global-header/components/UserContext.js

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
+export const UserContext = React.createContext(undefined);
```

Comment:
Or even like this since we are repeating the default user throughout files...

```suggestion

const defaultUser = {
  firstName: 'John',
  lastName: 'Doe',
  email: 'john.doe@hpe.com',
  image: '//s.gravatar.com/avatar/99020cae7ff399a4fbea19c0634f77c3?s=80',
};

export const UserContext = React.createContext({
  user: defaultUser,
  setUser: () => {},
});
```

Reply:
I think we still want to have the default user being set by the app so that the state can control/reset it to default, but I like the idea of providing the template like your above comment.

I've extracted the defaultUser variable to this file still and just imported it in the others for clarity/single source of code.